backend/eks/deploy.sh

```diff
@@ -28,9 +26,9 @@ workstation_cidr_block = "0.0.0.0/0"
 keypair_name = "$EKS_KEYPAIR"
 eks_version = "$EKS_VERS"
 cluster_labels = $EKS_CLUSTER_LABEL
-hosted_zone_id = "${R53_ZONE_ID}"
-hosted_zone_name = "${R53_ZONE_NAME}"
-hosted_zone_policy_arn = "${R53_ZONE_POLICY}"
+hosted_zone_id = "${EKS_ZONE_ID}"
```

Comment:
I think you need to put in some default values in defaults.sh or error out. All previous TF variables have default values.

Reply:
These DNS variables must be set by the user, and not setting them AFAIK does error out.